### PR TITLE
renovate: disable major version updates

### PR DESCRIPTION
**Pull request by rosstimothy**

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base"
+    "config:recommended",
```
Comment (codingllama) [inline .github/renovate.json:4]: Any other significant changes we should expect from this?

File: .github/renovate.json [hunk 2]
```diff
@@ -13,97 +14,97 @@
...
     },
     {
       "matchManagers": ["gomod"],
       "postUpdateOptions": ["gomodTidy"],
       "enabled": true,
-      "matchFiles": ["assets/aws/go.mod"],
+      "matchFileNames": ["assets/aws/go.mod"],
       "groupName": "Assets - AWS ",
-      "excludePackageNames": [
+      "excludematchPackageNames": [
```
Comment (codingllama) [inline .github/renovate.json:67]: Is it `excludematch` all lowercase? (Just checking.)

**Review by codingllama — APPROVED** [review]
Thanks!

File: .github/renovate.json (hunk 1 above)
Comment (codingllama) [reply .github/renovate.json:4]: It's just curiosity, I can lookup the docs later too.

**Review by zmb3 — APPROVED** [review]

**Review by jentfoo — APPROVED** [review]

**Pull request merged**